runtime-testsuite/resources/org/antlr/v4/test/runtime/templates/CompositeParsers/DelegatorInvokesDelegateRuleWithArgs.stg

```diff
@@ -28,5 +28,5 @@ WS : (' '|'\n') -> skip ;
 
 slaveGrammar(grammarName) ::= <<
 parser grammar S;
-a[int x] returns [int y] : B {<write("\"S.a\"")> $y=1000;} ;
```

Comment:
:memo: I'm looking into this a bit more. We have tests that contain duplicate actions (e.g. **MultipleActions.stg**), but they use empty actions. This was an oversight in the current tests since I always expected users would have the ability to include multiple consecutive actions in either a lexer or parser. I'll have more information tonight when I can next run the Python tests.